README.md

```diff
@@ -24,7 +24,7 @@ Installation
 
        composer install --no-dev
 
-   This will create a `config.php` file that you can edit.
+   Then create a `.env.local` file.
```

Comment:
Is there a reason why docker and symfony can't share the same .env file?

Reply:
I see. I do not want to stop this PR because of docker unless someone other than me is using it. @cimurah @MusikAnimal what do you think? Should we fix docker in a separate patch? Do you want to support it at all?